### PR TITLE
[IMP] Tests added

### PR DESCRIPTION
**Pull request by JavierIniesta**
Tests for product_variant_cost_price.

### CODE REVIEW

File: product_variant_cost_price/tests/__init__.py [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# -*- coding: utf-8 -*-
+# License AGPL-3: Antiun Ingenieria S.L. - Javier Iniesta
+# See README.rst file on addon root folder for more details
```
Comment (pedrobaeza) [inline product_variant_cost_price/tests/__init__.py]: Change this to the standard header: https://github.com/OCA/maintainer-tools/blob/master/template/module/__init__.py#L3
`# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).`

File: product_variant_cost_price/tests/test_product_product.py [hunk 2]
```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
+# License AGPL-3: Antiun Ingenieria S.L. - Javier Iniesta
+# See README.rst file on addon root folder for more details
```
Comment (pedrobaeza) [inline product_variant_cost_price/tests/test_product_product.py]: The same

File: product_variant_cost_price/tests/test_product_product.py [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+from openerp.tests.common import TransactionCase
+
+
+class TestProductProduct(TransactionCase):
+
+    def test_get_selection_cost_method(self):
+        self.env['product.product']._get_selection_cost_method()
+
+    def test_default_cost_method(self):
```
Comment (pedrobaeza) [inline product_variant_cost_price/tests/test_product_product.py]: This doesn't test what the module does

Comment (pedrobaeza) [inline product_variant_cost_price/tests/test_product_product.py]: Sorry, I remembered bad. It's correct to put this.

**Comment by pedrobaeza** [comment]
To exercise a bit more this. You have to create one product with multiple variants, and see that each one can have different cost prices.

**Comment by JavierIniesta** [comment]
License info changed and product variant cost prices test added.

**Pull request merged**